ripe/atlas/tools/commands/base.py

```diff
@@ -47,6 +48,28 @@ def __init__(self, *args, **kwargs):
         )
         self.user_agent = self._get_user_agent()
 
+    @staticmethod
+    def get_available_commands():
+        """
+        Get a list of commands that we can execute.  By default, we have a
+        fixed that we make available in this directory, but the user can create
```

Comment:
"we have a fixed that" -- probably supposed to be "fixed list"?  This is probably my typo :-)